### PR TITLE
Initial commit

### LINKED ISSUE #1: Create a pull request

### PR DESCRIPTION
**Pull request by SlothOfAnarchy**
fixes #1

### CODE REVIEW

File: files/ldap.conf [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#
+# LDAP Defaults
+#
+
+# See ldap.conf(5) for details
+# This file should be world readable but not world writable.
+
+#BASE	dc=example,dc=com
```
Comment (dasJ) [inline files/ldap.conf]: Please remove the useless example comments

File: meta/main.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  platforms:
+    - name: Ubuntu
+      versions:
+        - xenial
+        - yakkety
+        - zesty
+        - artful
+
+  galaxy_tags:
+    - stuvus
```
Comment (dasJ) [inline meta/main.yml]: Aren't there more tags? And why is there a stuvus tag? Literally nobody will ever search that

File: tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+- name: Copy LDAP-Client config
+  copy:
+    src: ldap.conf
+    dest: /etc/ldap/ldap.conf
+    owner: root
+    group: root
+    mode: "u=rw,g=r,o=r"
```
Comment (dasJ) [inline tasks/main.yml]: Why not the decimal notiation?

**Review by dasJ — CHANGES_REQUESTED** [review]

File: meta/main.yml (hunk 2 above)
Comment (SlothOfAnarchy) [reply meta/main.yml]: Please change this in ansible-skeleton if you don't agree with it.

Comment (dasJ) [reply meta/main.yml]: Okay

File: tasks/main.yml (hunk 3 above)
Comment (SlothOfAnarchy) [reply tasks/main.yml]: You're right, this stems from a wierd template.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+None.
+
+## Role Variables
+
+None.
+
+## Example Playbook
+
+Including an example of how to use your role (for instance, with variables passed in as parameters) is always nice for users too:
```
Comment (dasJ) [inline README.md]: Please remove this line

File: README.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+```
+
+## License
+
+This work is licensed under a [Creative Commons Attribution-ShareAlike 4.0 International License](https://creativecommons.org/licenses/by-sa/4.0/).
+
+
+## Author Information
+
+* [Markus Mroch (Mr. Pi)](https://github.com/Mr-Pi) _markus.mroch@stuvus.uni-stuttgart.de_
```
Comment (dasJ) [inline README.md]: Let me do a PR for that

**Review by dasJ — CHANGES_REQUESTED** [review]
Almost

File: README.md (hunk 4 above)
Comment (SlothOfAnarchy) [reply README.md]: Done.

**Review by dasJ — APPROVED** [review]

**Pull request merged**